src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -37,13 +39,26 @@ def get(self, request, project, version):
         except Release.DoesNotExist:
             raise ResourceDoesNotExist
 
+        ids_from_GroupResolution = GroupResolution.objects.filter(
```

Comment:
nitpick: these names are mixing snake case and camelcase, which is a bit weird. I'd just do this to simplify everthing:

```python
group_ids = set()
group_ids |= set(GroupResolution.objects.filter(...))
group_ids |= set(GroupCommitResolution.objects.filter(...))
```

Then you can simply pass this set along to the following queryset and avoid `chain()` as well, since that doesn't help you iwht much here. set() would also dedupe group ids.